01_fib/jimbotech/ex-fib_test.go

```diff
@@ -0,0 +1,46 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestFib(t *testing.T) {
```

Comment:
It would be worthwhile to test the boundary cases such as `fib(0)` and `fib(1)` and even better larger fib number.

Reply:
Done